src/components/glossarypage.tsx

```diff
@@ -26,10 +25,26 @@ const Glossary = () => {
     const termToDisplay = terms.filter((item) => item.term === obj); 
     setModalTerm(termToDisplay); 
   }
-
-  useEffect(() => {
+  //When page first loads display term with letter A/or when search input is empty
+  const displayAList = () => {
     const displayATerms = terms.filter((item) => item.term[0].toLowerCase() === 'a'); 
     setDisplayTerms(displayATerms); 
+  }
+
+  // SEARCH THROUGH GLOSSARY TERMS
+  const handleChange = (event) => {
+    setActiveBtn(''); 
+    const searchTerms = terms.filter((item) => item.term.toLowerCase().includes(event.target.value.toLowerCase()));  
+    setDisplayTerms(searchTerms); 
+
+    if(!event.target.value || event.target.value === ' ') {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/OutBoulderCounty/icc-ui/pullRequest?prid=8518374)
```suggestion
    if(!event.target.value || event.target.value === " ") {
```